.github/scripts/findUnusedStyles.ts

```diff
@@ -0,0 +1,462 @@
+        // Look for object literal properties (style definitions)
+        if (ts.isPropertyAssignment(node) || ts.isMethodDeclaration(node)) {
+            const key = this.extractKeyFromPropertyNode(node);
+
+            if (key && this.isStyleKey(key)) {
+                try {
+                    let shouldCapture = false;
+                    if (ts.isPropertyAssignment(node)) {
+                        // For PropertyAssignment, check if the value is an object literal
+                        if (ts.isObjectLiteralExpression(node.initializer)) {
+                            shouldCapture = true;
+                        }
+                    } else if (ts.isMethodDeclaration(node)) {
+                        // For MethodDeclaration, it's a function that returns styles
+                        shouldCapture = true;
+                    }
+
+                    if (shouldCapture) {
+                        const {line, character} = sourceFile.getLineAndCharacterOfPosition(node.getStart());
+                        this.styleDefinitions.set(key, {
+                            key,
+                            file,
+                            line: line + 1,
+                            column: character + 1,
+                        });
+                    }
+                } catch (error) {
+                    console.warn(`Warning: Could not get position for style '${key}' in ${file}`);
+                }
+            }
+        }
+
+        // Also look for variable declarations that might be styles
+        if (ts.isVariableDeclaration(node) && ts.isIdentifier(node.name)) {
+            const key = node.name.text;
+            if (this.isStyleKey(key)) {
+                try {
+                    const {line, character} = sourceFile.getLineAndCharacterOfPosition(node.getStart());
+                    this.styleDefinitions.set(key, {
+                        key,
+                        file,
+                        line: line + 1,
+                        column: character + 1,
+                    });
+                } catch (error) {
+                    console.warn(`Warning: Could not get position for style '${key}' in ${file}`);
+                }
+            }
+        }
+    }
+
+    private isStyleKey(key: string): boolean {
+        // Skip certain patterns that are likely not style keys
+        if (STYLE_KEY_SKIP_PATTERNS.includes(key)) {
+            return false;
+        }
+
+        // Must be a valid identifier
+        if (!VALID_IDENTIFIER_PATTERN.test(key)) {
+            return false;
+        }
+
+        return true;
+    }
+
+    private loadAllFileContents(): void {
+        // Load all source files EXCEPT style definition files
+        const allFilesPaths = `src/**/${STYLE_FILE_EXTENSIONS}`;
+        const allFiles = globSync(allFilesPaths, {
+            cwd: this.rootDir,
+        });
+
+        // Filter out style definition files, BUT keep utils/generators/themes for usage checking
```

Comment:
Similar comment as above - seems like you could achieve this more simply with `ignore` param for `globSync`.